tests/test_installer_backend.py

```diff
@@ -221,16 +247,57 @@ def test_do_configure(tmpdir):
 
 def test_do_aws_configure(tmpdir, monkeypatch):
     monkeypatch.setenv('BOOTSTRAP_VARIANT', 'test_variant')
+    create_fake_build_artifacts(aws_base_config, tmpdir)
+
+    with tmpdir.as_cwd():
+        assert backend.do_aws_cf_configure() == 0
+
+
+valid_storage_config = """---
+master_list:
+ - 127.0.0.1
+aws_template_storage_access_key_id: {}
```

Comment:
Would be good to use named parameters in the format string here to make it more readable what is going where (And easier to extended in the future)

Rather than `{}` can do `{name}` and then `.format(name="foobar")`